VALIDATORS_FAQ.md

```diff
@@ -145,47 +144,47 @@ We have to solve this simple equation to find the reward R for each validator:
 `9*R + R + R*5% = 1005 ⇔ R = 1005/10.05 = 100`
 
 * For the proposer validator
-  * The pool obtains R + R*5% = 105 Atoms
-  * Commission = 105*80%*1% = 0.84 Atoms
-  * Validator reward = 100 * 20% + Commission = 21.84 Atoms
-  * Delegators rewards = 105 * 80% - Commission = 83.16 Atoms (each delegator will be able to claim its portion of these rewards in proportion of their stake)
+  * The pool obtains R + R*5%: 105 Atoms
+  * Commission: 105*80%*1% = 0.84 Atoms
+  * Validator reward: 100 * 20% + Commission = 21.84 Atoms
+  * Delegators rewards: 105 * 80% - Commission = 83.16 Atoms (each delegator will be able to claim its portion of these rewards in proportion to their stake)
 * For each non-proposer validator
-  * The pool obtains R = 100 Atoms
-  * Commission = 100*80%*1% = 0.8 Atoms
-  * Validator reward = 100 * 20% + Commission = 20.8 Atoms
-  * Delegators rewards = 100 * 80% - Commission = 79.2 Atoms (each delegator will be able to claim its portion of these rewards in proportion of their stake)
+  * The pool obtains R: 100 Atoms
+  * Commission: 100*80%*1% = 0.8 Atoms
+  * Validator reward: 100 * 20% + Commission = 20.8 Atoms
+  * Delegators rewards: 100 * 80% - Commission = 79.2 Atoms (each delegator will be able to claim its portion of these rewards in proportion to their stake)
 
 ### What are the slashing conditions?
 
-If a validator misbehaves, its bonded stake along with its delegator’s stake and will be slashed. The severity of the punishment depends on the type of fault. There are 3 main faults that can result in loss of funds for a validator and its delegators :
+If a validator misbehaves, its bonded stake along with its delegators' stake and will be slashed. The severity of the punishment depends on the type of fault. There are 3 main faults that can result in slashing of funds for a validator and its delegators:
 
 * **Double signing:** If someone reports on chain A that a validator signed two blocks at the same height on chain A and chain B, this validator will get slashed on chain A
 * **Unavailability:** If a validator’s signature has not been included in the last X blocks, the validator will get slashed by a marginal amount proportional to X. If X is above a certain limit Y, then the validator will get unbonded
 * **Non-voting:** If a validator did not vote on a proposal and once the fault is reported by a someone, its stake will receive a minor slash. 
 
 Note that even if a validator does not intentionally misbehave, it can still be slashed if its node crashes, looses connectivity, gets DDOSed, or if its private key is compromised.
```

Comment:
Isn't this spelt differently normally?

Reply:
Also, on the first use let's use the written out name and then abbreviate.

Reply:
What is ?

Reply:
nvm, I don't remember :-)